### DISCUSSION TITLE
Havoc DH Rotation

### DISCUSSION THREAD

**Discussion by Misclicker** [Hekili/hekili#1024] [Q&A] [answered]
Hi Hekili,

Thank you for creating this, its a great help for us to learn our rotation.

Just a quick question, I am wondering if I am able to incorporate more abilities into my bar with the addon? It would be nice if I could add both my on use trinkets in there and some of my other abilities, I know it may not be possible but just thought I'd ask.

Thanks

**Comment by Hekili** [comment, marked as answer]
You can incorporate any addon-supported abilities in a custom priority.  The FAQ has a little bit of information about that.
